pkg/proxy/ipvs/proxier.go

```diff
@@ -1882,13 +1882,13 @@ func (proxier *Proxier) createAndLinkeKubeChain() {
 			if chain, ok := existingNATChains[ch.chain]; ok {
 				writeBytesLine(proxier.natChains, chain)
 			} else {
-				writeLine(proxier.natChains, utiliptables.MakeChainLine(kubePostroutingChain))
+				writeLine(proxier.natChains, utiliptables.MakeChainLine(ch.chain))
```

Comment:
@andrewsykim can we have the same issue we had here https://github.com/kubernetes/kubernetes/pull/94591, that is cleaning rules?

Reply:
I had thought defaulting to `KUBE-POSTROUTING` here is intentional? Either way @hanlins I don't think we want to make this change in this PR.